### DISCUSSION TITLE
any authentication method.

### DISCUSSION THREAD

**Discussion by edib** [rapidloop/rapidrows#2] [General]
It seems there is no authentication method. What is your plan?

**Comment by mdevan** [comment]
What kind of auth method are you looking for?

In general, there is a roadmap concept of having a "middleware" or "hook" script that can intercept endpoints of a specified pattern before they are sent to their defined handlers.

**Comment by edib** [comment]
I have used "postgrest" earlier, it has a predefined jwt authentication method. I ment simply that kind of authentication.
